### ISSUE TITLE
Implement paging with page size, before and after

### ISSUE THREAD

**Issue by AndyCC** [AndyCC/EfCore.Filtering#5]

**Comment by Deathrage** [comment]
Is this cursor pagination? If so I'd like to make the same feature request.

**Comment by AndyCC** [comment]
Yes it will be an implementation of something similar to https://jsonapi.org/profiles/ethanresnick/cursor-pagination/
However I've not got much time to work on this at the moment
